Source/Core/Core/HW/SI/SI.cpp

```diff
@@ -381,7 +409,9 @@ void Init()
     {
       s_desired_device_types[i] = SIDEVICE_NONE;
 
-      if (Movie::IsUsingPad(i))
+      if (Movie::IsUsingGBA(i))
+        s_desired_device_types[i] = SIDEVICE_GC_GBA;
```

Comment:
We prefer balanced curlies, and since the `else` has some the `if` should have them as well.
```suggestion
      if (Movie::IsUsingGBA(i))
      {
        s_desired_device_types[i] = SIDEVICE_GC_GBA;
      }
```